### PR TITLE
[IMP] MRP: Backorders Shop Floor workflow

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR removes the tablet view section from the Manufacturing backorders doc, and replaces it with a section about the Shop Floor workflow.

### CODE REVIEW

File: content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst [hunk 1]
```diff
@@ -63,127 +65,67 @@ order.
...
 
-The steps for the rest of the workflow depend on whether the manufacturing order being processed
-requires the completion of a single work order or multiple work orders.
+Complete the steps on the work order's card until the :guilabel:`Register Production` step is
+reached, and then click on it to open the :guilabel:`Register Production` pop-up window.
 
-Single work order
------------------
+.. important::
+   Do not click the :guilabel:`# Units` button on the right side of the step. Doing so will
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst:95]: To emphasize
```suggestion
   Do **not** click the :guilabel:`# Units` button on the right side of the step. Doing so will
```

File: content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst [hunk 2]
```diff
@@ -63,127 +65,67 @@ order.
...
-create one by clicking :guilabel:`Create`.
-
-If a new manufacturing order is created, select a product from the :guilabel:`Product` drop-down
-menu and enter a quantity of two or more in the :guilabel:`Quantity` field, then click
-:guilabel:`Confirm` to confirm the order.
-
-After confirming the manufacturing order, select the :guilabel:`Work Orders` tab and click the
-:guilabel:`📱 (tablet view)` button located on the line of the first work order to enter the tablet view.
+   In order to use the *Shop Floor* module, the *Work Orders* setting must be enabled. To do so,
+   navigate to :guilabel:`Manufacturing --> Configuration --> Settings`, enable the :guilabel:`Work
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst:75]:
```suggestion
   navigate to :menuselection:`Manufacturing --> Configuration --> Settings`, enable the :guilabel:`Work
```

**Review by Felicious — APPROVED** [review]
Great job with this doc, @hojo-odoo !

There was a good mix of images and admonition blocks to make this doc an engaging read. I also really liked the dynamic image on line 105 where the WO line pops out into the Register Production pop-up. It really gives the reader the context where that pop-up window comes from.

Great job with this!!

File: content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst [hunk 3]
```diff
@@ -63,127 +65,67 @@ order.
...
-
-If a new manufacturing order is created, select a product from the :guilabel:`Product` drop-down
-menu and enter a quantity of two or more in the :guilabel:`Quantity` field, then click
-:guilabel:`Confirm` to confirm the order.
-
-After confirming the manufacturing order, select the :guilabel:`Work Orders` tab and click the
-:guilabel:`📱 (tablet view)` button located on the line of the first work order to enter the tablet view.
+   In order to use the *Shop Floor* module, the *Work Orders* setting must be enabled. To do so,
+   navigate to :menuselection:`Manufacturing --> Configuration --> Settings`, enable the
+   :guilabel:`Work Orders` checkbox, and then click :guilabel:`Save`.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst:76]:
```suggestion
   In order to use the *Shop Floor* module, the *Work Orders* setting **must** be enabled.
   
   To do so, navigate to :menuselection:`Manufacturing --> Configuration --> Settings`, enable the
   :guilabel:`Work Orders` checkbox, and click :guilabel:`Save`.
```

File: content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst [hunk 4]
```diff
@@ -63,127 +65,67 @@ order.
...
+(square with arrow coming out of it)` button on the line of the work order to be processed. On the
+resulting :guilabel:`Work Orders` pop-up window, click the :guilabel:`Open Shop Floor` button to
+open the *Shop Floor* module.
 
-.. image:: manufacturing_backorders/units-field.png
-   :align: center
-   :alt: The Units field in the tablet view.
+When accessed from a specific work order, the *Shop Floor* module opens to the page for the work
+center where the order is configured to be processed, and isolates the work order's card so that no
+other cards are shown.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst:89]:
```suggestion
center where the order is configured to be processed, and isolates the work order's card, so no
other cards are shown.
```

File: content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst [hunk 5]
```diff
@@ -63,127 +65,67 @@ order.
...
-The steps for the rest of the workflow depend on whether the manufacturing order being processed
-requires the completion of a single work order or multiple work orders.
+Complete the steps on the work order's card until the :guilabel:`Register Production` step is
+reached, and then click on it to open the :guilabel:`Register Production` pop-up window.
 
-Single work order
------------------
+.. important::
+   Do **not** click the :guilabel:`# Units` button on the right side of the step. Doing so will
+   automatically register the full amount of units as having been produced.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst:96]:
```suggestion
   Do **not** click the :guilabel:`# Units` button on the right side of the step. Doing so
   automatically registers the full amount of units as having been produced.
```

File: content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst [hunk 6]
```diff
@@ -63,127 +65,67 @@ order.
...
 
-If the manufacturing order only requires the completion of a single work order, complete the work
-order, then click :guilabel:`Mark As Done And Close MO`. The manufacturing order is closed and a
-backorder for the units that still need to be manufactured is created automatically.
+On the :guilabel:`Register Production` pop-up window, enter the number of units produced in the
+:guilabel:`Quantity` field. Make sure that the number entered is less than the number of units
+listed to the right of the field. Then, click :guilabel:`Validate`.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst:100]:
```suggestion
:guilabel:`Quantity` field. Make sure the number entered is *less* than the number of units listed
to the right of the field. Then, click :guilabel:`Validate`.
```

File: content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst [hunk 7]
```diff
@@ -63,127 +65,67 @@ order.
...
-Done And Close MO` after completing all of the work orders.
+   :alt: The Register Production pop-up window in the Shop Floor module.
 
-If only some of the remaining units will be manufactured immediately, enter the number in the
-:guilabel:`Units` field at the top left of the tablet view, then click :guilabel:`Record Production`
-to create an additional backorder for the remaining units, with *003* at the end of its reference
-tag.
+The pop-up window disappears and the :guilabel:`# Units` button on the work order's card updates to
+reflect the number of units produced, as a fraction of the number of units the |MO| was originally
+created for.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst:108]:
```suggestion
The pop-up window disappears, and the :guilabel:`# Units` button on the work order's card updates to
reflect the number of units produced, as a fraction of the number of units for which the |MO| was
originally created.
```

File: content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst [hunk 8]
```diff
@@ -63,127 +65,67 @@ order.
...
+Next, click the :guilabel:`Mark as Done` button at the bottom-right of the work order's card. The
+work order card begins to fade away. Once it disappears completely, a new work order card appears,
+titled with the original |MO|'s reference number with a `-002` tag added to the end of it.
 
-.. tip::
-   It is also possible to create a backorder in the middle of a manufacturing order, when some but
-   not all of the work orders have already been completed. Doing so marks the completed work
-   order(s) as :guilabel:`Finished` on the backorder.
+This new reference number represents the backorder |MO|. The original |MO|'s reference number will
+now appear with a `-001` tag added to the end of it to distinguish it from the backorder |MO|.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst:115]:
```suggestion
This new reference number represents the backorder |MO|. The original |MO|'s reference number now
appears with a `-001` tag added to the end of it to distinguish it from the backorder |MO|.
```

**Review by ksc-odoo — APPROVED** [review]
@hojo-odoo Another solid doc! Great work. Just a couple small things for you to address, but once those are taken care of, I think this one will be ready for the next stage! 👍

**Review by samueljlieber — APPROVED** [review]
Hi @hojo-odoo! Nice job on this PR, everything looks good to me! I am approving now with the ask to shorten the commit message & PR title to be more concise, something like:
```
[IMP] mrp: backorders shop floor workflow
```

**Comment by hojo-odoo** [comment]
@StraubCreative This is ready for merge.

File: content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst [hunk 9]
```diff
@@ -63,127 +65,67 @@ order.
...
-   :align: center
-   :alt: The tablet view button for a work order on a manufacturing order.
+To create a backorder from the *Shop Floor* module, begin by navigating to
+:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`. Select an |MO| for multiple
+units of a product, for which a backorder needs to be created.
 
-Once in tablet view, enter the quantity being manufactured immediately in the :guilabel:`Units`
-field at the top left of the tablet view.
+On the |MO|, select the :guilabel:`Work Orders` tab, and then click the :guilabel:`Open Work Order
+(square with arrow coming out of it)` button on the line of the work order to be processed. On the
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst:83]:
```suggestion
(external link icon)` button on the line of the work order to be processed. On the
```

**Review by StraubCreative — APPROVED** [review]
@robodoo delegate=hojo-odoo

**Comment by hojo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**